server/StudySharp.API/Responses/TheoryBlocks/GetTheoryBlocksResponse.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using StudySharp.Domain.Models;
+
+namespace StudySharp.API.Responses.TheoryBlocks
+{
+    public class GetTheoryBlocksResponse
```

Comment:
Theory blocks can only exist in the scope of course. This file is redundant. Delete this file